apps/system/js/accessibility.js

```diff
@@ -284,6 +284,22 @@
     },
 
     /**
+     * Start a timeout that waits to display hints
+     * @memberof Accessibility.prototype
+     */
+     setHintsTimeout: function ar_setHintsTimeout(hints) {
+      var accessibilityObj = this;
```

Comment:
No need to retaining this here. just use the format of:
```setTimeout(function() {...}.bind(this), ...); ```
.bind lets you set 'this' for the function scope.